javacord-core/src/main/java/org/javacord/core/util/handler/message/MessageCreateHandler.java

```diff
@@ -98,6 +105,20 @@ private void handle(TextChannel channel, JsonNode packet) {
                 author.asUser().orElse(null),
                 author.isWebhook() ? author.getId() : null,
                 event);
+
+        optionalReferencedMessage.ifPresent(referencedMessage -> {
```

Comment:
```suggestion
        message.getReferencedMessage().ifPresent(referencedMessage -> {
```
And remove the variable assignment in line 93